src/views/v2/month/calendar-body/day/calendar-events/calendar-event/title.php

```diff
@@ -20,13 +20,16 @@
 <h3 class="tribe-events-calendar-month__calendar-event-title tribe-common-h8 tribe-common-h--alt">
 	<a
 		href="<?php echo esc_url( $event->permalink ) ?>"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
```

Comment:
https://github.com/moderntribe/the-events-calendar/pull/2987/files?utf8=%E2%9C%93&diff=split&w=1#r366022101